src/App.tsx

```diff
@@ -1,61 +1,98 @@
 import './App.scss';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [errorVisiableUser, setErrorVisiableUser] = useState(false);
+  const [errorVisiableTitle, setErrorVisiableTitle] = useState(false);
+
+  const handleSuccess = (userId: number) => {
+    if (userId !== 0 && newTodoTitle !== '') {
+      setTodos([
+        ...todos,
+        {
+          userId,
+          id: todos.length + 1,
+          title: newTodoTitle,
+          completed: false,
+        },
+      ]);
+
+      setSelectedUserId(0);
+      setNewTodoTitle('');
+    }
+
+    if (userId === 0) {
+      setErrorVisiableUser(true);
+    } else {
+      setErrorVisiableUser(false);
+    }
+
+    if (newTodoTitle === '') {
```

Comment:
Why did you set errors to false? It's default value. You need set it on inputs changes (not on submit)
```suggestion
    if (!userId === 0) {
      setErrorVisiableUser(true);
    }

    if (!newTodoTitle) {
```